packages/codemod_riverpod/bin/codemod_riverpod.dart

```diff
@@ -1,11 +1,48 @@
+import 'dart:io';
+
 import 'package:codemod/codemod.dart';
 import 'package:codemod_riverpod/codemod_riverpod.dart';
 import 'package:glob/glob.dart';
+import 'package:pub_semver/pub_semver.dart';
+import 'package:pubspec_parse/pubspec_parse.dart';
+
+Future<void> main(List<String> args) async {
+  const latestVersion = '0.14.0';
+  final pubspecFile = File('pubspec.yaml');
+  if (!pubspecFile.existsSync()) {
+    stderr.writeln(
+        'Pubspec not found! Are you in the root directory of your package / app?');
+    return;
+  }
+  HostedDependency dep;
```

Comment:
I'd just do:
```dart
final dev = pubspec.dependencies['hooks_riverpod'] ?? pubspec.dependencies['flutter_riverpod'] ?? pubspec.dependencies['riverpod'];
```

Also, we need to handle dependencies other than `HostedDependency`, such as path or git dependencies.
One use-case is the `examples` folder, which should be supported. But it's also possible that some apps use a fork of riverpod

In those cases, I'd just run all migrations (since they should be no-op if the code is already migrated)

Reply:
Okay, there would be additional work to make them no-ops if the code is already migrated. But I'll work on that.